src/utils/sentry.ts

```
@@ -18,13 +18,43 @@ export class SentryTransport {
+      };
 
-      this.sentry.captureMessage(data, this.getSentryLevel(levelName));
+      this.sentry.addBreadcrumb(breadcrumb);
     } catch (err) {
       console.error("Sentry transport error:", err);
     }
   }
 
+  private formatData(log: Record<string, unknown>): Record<string, unknown> {
+    const data = { ...log };
+    delete data.level;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)